content/applications/sales/crm/track_leads/unattended_leads_report.rst

```diff
@@ -0,0 +1,180 @@
+:guilabel:`Due Date` to add it to the rule.
+
+  .. image:: unattended_leads_report/activities-due.png
+     :align: center
+     :alt: Custom filter pop-up with emphasis on the options for activities and due date.
+
+Then, click into then next field and select :guilabel:`<=` from the drop-down menu. Selecting this
+operator includes all activities with a due date up to, and including, the date selected in the next
+field.
+
+The third field can be left as today's date, or adjusted as needed.
+
+Exclude unassigned leads
+------------------------
+
+Click :guilabel:`New Rule`. Click the first field for the new rule, and type `Salesperson` in the
+:guilabel:`Search...` bar, or scroll to search through the list to locate it.
+
+Click in the rule's second field, then select :guilabel:`is set` from the drop-down menu. Selecting
+this operator excludes any leads not assigned to a specific salesperson.
+
+Add a Sales team
+----------------
+
+Click :guilabel:`New Rule`. Click the first field for the new rule, and type `Sales Team` in the
+:guilabel:`Search...` bar, or scroll to search through the list to locate it.
+
+Click in the rule's second field, select :guilabel:`is in` from the drop-down menu. Selecting this
+operator limits results to the sales teams selected in the next field.
+
+In the third field, select the desired sales team from the drop-down menu. Multiple teams can be
+selected in this field.
+
+.. note::
+   This filter is optional. To view results for the entire company, do **not** add this filter.
+
+.. figure:: unattended_leads_report/configured-custom-rules.png
+   :align: center
+   :alt: An example of the Custom Filter pop-up window with all of the rules configured.
+
+   An example of the **Add Custom Filter** pop-up window with all of the rules configured.
+
+View results
+------------
+
+At the top of the :guilabel:`Add Custom Filter` form, there is an option to match :guilabel:`any` or
+:guilabel:`all` of the rules. In order to properly run the report, only records that match **all**
+of the following filters should be included. Before adding the filters, make sure :guilabel:`all` is
+selected in this field.
+
+.. image:: unattended_leads_report/all-custom-filter.png
+   :align: center
+   :alt: Example of overdue activities notification in the chatter of a lead.
+
+After the filters are configured, click :guilabel:`Add`. The resulting report displays all leads
+assigned to a salesperson where an activity is past due, or is due on the current date. The default
+display is a bar graph, where the leads are grouped by *stage*.
+
+To group the results by salesperson, click the down arrow to the right of the search bar. Under the
+:guilabel:`Group By` heading, select :guilabel:`Salesperson`.
+
+.. note::
+   The option to group by :guilabel:`Sales Team` is also available under the :guilabel:`Group By`
+   heading.
+
+To change to a *list* view, click the :guilabel:`≣ (bars)` icon in the top-right corner of the
```

Comment:
Is **(bars)** what that icon is actually called? 
Hint 👇 
![Screenshot 2024-03-11 at 3 08 31 PM](https://github.com/odoo/documentation/assets/5496809/590ec04c-c142-417a-8f2d-fc8f6487b818)